app/posts/manage-teacher-training-applications/2020-12-18-search-by-candidate-name-or-reference.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Search by candidate name or reference
+description: Let users search the applications list by candidate name or reference
+date: 2020-12-18
+---
+
+We updated the search to allow users to find applications by reference in addition to candidate name.
+
+The search works independently from the filters, allowing users to filter or search in whatever way suits their needs.
+
+This change included:
+
+- moving the search form to above the applications list to accept terms that are compared to a candidate’s full name or reference
+- removing the candidate name search from the filters side bar
```

Comment:
Why did we do that?

Reply:
Also this is really a dupe of the first bullet.

Reply:
De-duped.